### ISSUE TITLE
(?) Is there any way to change column dendrogram height

### ISSUE THREAD

**Issue by Mesya82** [skutac/InCHlib.js#22]
So as the title says. Vertical dendrogram(not column) I can manipulate by changing heatmap_part_width. As I can see right now, there's no way I can change height of column_dendrogram. Or am I missing something?

**Comment by skutac** [comment]
You are right. Currently, there is no way how to adjust column dendrogram size. I'll try to add more intelligent (adjustable) way how to set the size of different cluster heatmap parts.

**Comment by Mesya82** [comment]
That would be great!
